layout-management-webapps/src/main/webapp/vue-app/site-navigation/components/SiteNavigationAddNodeDrawer.vue

```diff
@@ -154,6 +163,14 @@ export default {
       ],
     };
   },
+  computed: {
+    enableNextBtn(){
+      return this.nodeId && this.nodeLabel;
```

Comment:
the next button should not be enabled when nodeId is typed with unauthorized chars